### PR TITLE
Validate $YOLO

### PR DESCRIPTION
**Pull request by nolomax**
# Validate $[YOLO](https://solscan.io/token/4jE4VuqFWZfmSXjTDD5KUoN2qkVu96nPTPj4mhs5PA1W)

## Attestations:
- Tweet from your Twitter Account attesting the Mint address, tagging [@JupiterExchange](https://twitter.com/JupiterExchange) and showing community support: https://twitter.com/YoloNoloBot/status/1666596398169563138?s=20
- CMC URL: https://coinmarketcap.com/currencies/yolonolo/

## Validation (Please check off boxes):
- [x] The metadata provided in the PR matches what is on-chain (Mandatory)
- [x] Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
- [x] Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)  

## Acknowledgement (Please check off boxes)
- [x] My change matches the format in the file (no spaces between fields).
- [x] My token is already live and trading on Jupiter.
- [x] !!! I read the README section on Community-Driven Validation and understand this PR will be only be reviewed when there is community support on Twitter.

### CODE REVIEW

**Comment by cinojosa0705** [comment]
I attest that the following information is correct:

[x] Mint Address: 4jE4VuqFWZfmSXjTDD5KUoN2qkVu96nPTPj4mhs5PA1W
[x] Symbol: YOLO
[x] Token Decimals: 6
[x] Token Icon: https://nftstorage.link/ipfs/bafkreia5uq5dr4nvgtb7dhrspmyov435vok7scobx4kbpwp7jwkuio5hxq
[x] On-chain metadata matches
[x] No duplication on Jupiter's strict list

**Comment by tails3x** [comment]
I attest that the following information is correct:

[x] Mint Address: 4jE4VuqFWZfmSXjTDD5KUoN2qkVu96nPTPj4mhs5PA1W
[x] Symbol: YOLO
[x] Token Decimals: 5
[x] Token Icon: https://nftstorage.link/ipfs/bafkreia5uq5dr4nvgtb7dhrspmyov435vok7scobx4kbpwp7jwkuio5hxq
[x] On-chain metadata matches
[x] No duplication on Jupiter's strict list

File: validated-tokens.csv [hunk 1]
```diff
@@ -737,4 +737,5 @@ CULT,$CULT,Cu1tCSoauo4Vtqsr9cD86RS4XJqS88LauU69AFV6KMH7,9,https://arweave.net/m3
 Komoverse Token,KOMO,KomoNuvereGko9jNKUgb3AJZdG3pDaXjbph77ZWvFy1,2,https://komoverse.io/assets/img/favicon.png,true
 BEN,$BEN,2Ry7AGS1w488LdSxhz2XvSrU1EVUzUGJo27DBP57couh,7,https://arweave.net/vvRsvtg1dtVxoUL6XNeRp-cOIYp8BRbaDigmuKvLazk,true
 Aevum,AEVUM,EsirN3orp85uyvZyDrZnbe9cyo7N1114ynLFdwMPCQce,9,https://arweave.net/mqtxUgbHYOu4lEUzMZy5YFZtItQQZtjZWbFUIbcM6cU,true
-BonkEarn,BERN,CKfatsPMUf8SkiURsDXs7eK6GWb4Jsd6UDbs7twMCWxo,5,https://i.imgur.com/nd9AVZ4.jpeg,true
\ No newline at end of file
+BonkEarn,BERN,CKfatsPMUf8SkiURsDXs7eK6GWb4Jsd6UDbs7twMCWxo,5,https://i.imgur.com/nd9AVZ4.jpeg,true
+YoloNolo Coin,YOLO,4jE4VuqFWZfmSXjTDD5KUoN2qkVu96nPTPj4mhs5PA1W,5,https://nftstorage.link/ipfs/bafkreia5uq5dr4nvgtb7dhrspmyov435vok7scobx4kbpwp7jwkuio5hxq,true
```
Comment (Timoon21) [inline validated-tokens.csv:741]:
```suggestion
YoloNolo Coin,YOLO,4jE4VuqFWZfmSXjTDD5KUoN2qkVu96nPTPj4mhs5PA1W,5,https://nftstorage.link/ipfs/bafybeiepttutbes4mzojtmlvx3iqgwvs673czt5g7k7uoficx4vmjunmie/YOLO LOGO.png,true
```

**Review by Timoon21 — CHANGES_REQUESTED** [review]

**Review by Timoon21 — APPROVED** [review]
- [x] On-chain metadata matches the PR
- [x] Token address matches the tweet
- [x] No duplication on Jupiter's strict list
- [x] No duplication on CMC/CoinGecko
- [x] Community validation

**Pull request merged**